html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="mail">Email</label>
+      <p><input type="email" id="mail" /></p>
+      <label for="whyCode">Why is coding cool?</label>
+      <p>
+        <select>
+          <option selected>Select an option</option>
+          <option>Because it's interesting.</option>
+          <option>Because it's fun.</option>
+          <option>Because it's powerful.</option>
+        </select>
+      </p>
+      <p>Favorite Language (choose one)</p>
+      <input type="radio" id="html" name="language" />
+      <label for="html">HTML</label>
+      <input type="radio" id="css" name="language" />
+      <label for="css">CSS</label>
+      <input type="radio" id="JavaScript" name="language" />
+      <label for="javascript">JavaScript</label>
```

Comment:
The `id` for your `input` should be "javascript" instead of "JavaScript" to match your `for` attribute